addon/index.js

```diff
@@ -58,8 +48,16 @@ export {
   getPretender,
 };
 
+/**
+ * Setup and teardown code, intended to be called with qunit hooks so that it can run code before & after each test.
+ */
 export function setupFactoryGuy(hooks) {
   hooks.beforeEach(function () {
-    manualSetup(this);
+    const { owner } = getContext();
+    FactoryGuy.setStore(owner.lookup('service:store'));
+  });
+
+  hooks.afterEach(function () {
+    FactoryGuy.reset();
```

Comment:
Note that we're not calling `loadScenarios()` or `loadFactories()` anymore, avoiding `require()` altogether - it's up to the app to import what they want to use.

A migration guide or the changelog will detail the changes needed, but briefly;
* they'll want to import their factories before any tests run (likely in test-helper.js)
* they'll want to import any scenario(s) they want to use, and call run(), for each test (likely making use of beforeEach hooks)

See readme for more info

---

Will look into the store service deprecations in later work